tests/unit/Framework/MockObject/GeneratorTest.php

```diff
@@ -45,6 +45,41 @@ public function testGetMockFailsWhenInvalidFunctionNameIsPassedInAsAFunctionToMo
         $this->generator->getMock(stdClass::class, [0]);
     }
 
+    public function testGetMockFailsWithInvalidClass(): void
```

Comment:
Can we be more specific in the test method names what "invalid" means (e.g., final)?

Reply:
done